src/domain/hooks/hooks.repository.ts

```diff
@@ -167,6 +167,10 @@ export class HooksRepositoryWithNotifications implements IHooksRepository {
             chainId: event.chainId,
             address: event.address,
           }),
+          this.stakingRepository.clearStakes({
```

Comment:
Nit: should we also clear on `MODULE_TRANSACTION`?

Reply:
Good point - yes! A module could execute a staking transaction. Changed in 21667dc4b3e336a0428a1de3527941ffe555a92b.